easybuild/easyconfigs/a/ASAGI/ASAGI-1.0_fix-level.h.patch

```diff
@@ -0,0 +1,11 @@
+--- ASAGI.orig/grid/level/level.h	1970-01-01 01:00:00.000000000 +0100
```

Comment:
@boegel Please add a description to this patch.

Reply:
fixed in 518427fee7